templates/.github/workflows/fogg_ci.yml.tmpl

```diff
@@ -28,8 +28,8 @@ jobs:
             exit 1
           fi
 {{- if not (eq (len $githubActionsCI.AWSProfiles) 0) }}
-      - run: aws configure set aws_access_key_id     $IDACCT_AWS_ACCESS_KEY_ID     --profile _idacct
-      - run: aws configure set aws_secret_access_key $IDACCT_AWS_SECRET_ACCESS_KEY --profile _idacct
+      - run: aws configure set aws_access_key_id     {{`${{`}} secrets.IDACCT_AWS_ACCESS_KEY_ID {{`}}`}}     --profile _idacct
```

Comment:
I couldn't figure out a better way to put a literal `{{` and `}}` in a go text template. Open to suggestions.